lib/irida/pipeline.rb

```diff
@@ -58,6 +58,13 @@ def property_pattern(property_name)
       sample_sheet['items']['properties'][property_name]['pattern']
     end
 
+    def samplesheet_schema_overrides_for_entry(entry, version)
```

Comment:
Suggestion (blocking): Can you move this under the private section as this should only be called during object initialization.

Reply:
Moved under private scope in e165362d7e29ae127e011b58492019004f9b0c57